void/sniffer.py

```diff
@@ -29,46 +30,42 @@
 from astropy.time import Time
 from astropy.io import fits
 
+from void import common
 
 log = logging.getLogger(__name__)
 
 
-LOG_FORMAT = (
-    '%(levelname)-8s  %(asctime)s  %(process)-5d  %(name)-26s  %(message)s')
-
-
 class Sniffer:
     DISABLED_FLAG: str = '0'
 
     def __init__(
             self,
             search_dir: str,
             maxn: int,
-            range_str: str,
+            tmin: str,
+            tmax: str,
             flag_name: str):
         self.search_dir = search_dir
         self.maxn = maxn
-        self.range_str = range_str
+        self.tmin = tmin
+        self.tmax = tmax
         if flag_name == self.DISABLED_FLAG:
             self.flag_name = None
         else:
             self.flag_name = flag_name
         self.count = 0
 
         self._filter_method = self._filter_always_true
-        if self.range_str:
-            if ',' in self.range_str:
-                # TODO partial?
-                before_comma, after_comma = self.range_str.split(',')
-                self.time_first = self.parse_time(before_comma[1:])
-                self.time_last = self.parse_time(after_comma[1:-1])
-                self._filter_method = self._filter_within
-            elif self.range_str.startswith('>'):
-                self.time_thresh = self.parse_time(self.range_str[1:])
-                self._filter_method = self._filter_after
-            elif self.range_str.startswith('<'):
-                self.time_thresh = self.parse_time(self.range_str[1:])
-                self._filter_method = self._filter_before
+        if self.tmin and self.tmax:
+            self.time_first = self.parse_time(self.tmin)
+            self.time_last = self.parse_time(self.tmax)
+            self._filter_method = self._filter_within
+        elif self.tmin:
+            self.time_thresh = self.parse_time(self.tmin)
```

Comment:
This is good, but we can make it better still.
I think there is no need to do the assigning of the methods like before. We can simply prepare `self.time_first` and `self.tile_last` here, like so:
```python
 if self.tmin:
    self.time_first = self.parse_time(self.tmin)
if self.tmax:
    self.time_last = self.parse_time(self.tmax)
```

... and the we can simplify `filter_fits` to something similar to this:

```python
def filter_fits(self, fname_i):
    if self.time_first and self.time_first < fname_i:
        return False
    if self.time_last and self.time_last > fname_i:
        return False
    return True
```

... and delete all those `_filter_before`, `_filter_within`... methods.

The less code the better :)